login-workflow/src/new-architecture/screens/VerifyCodeScreen/VerifyCodeScreenBase.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useCallback } from 'react';
+import { VerifyCodeScreenProps } from './types';
+import { WorkflowCard } from '../../components/WorkflowCard';
+import { WorkflowCardActions } from '../../components/WorkflowCard/WorkflowCardActions';
+import { WorkflowCardBody } from '../../components/WorkflowCard/WorkflowCardBody';
+import { WorkflowCardHeader } from '../../components/WorkflowCard/WorkflowCardHeader';
+import { WorkflowCardInstructions } from '../../components/WorkflowCard/WorkflowCardInstructions';
+import TextField from '@mui/material/TextField';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+
+/**
+ * Component that renders a screen that prompts a user to enter the confirmation code
+ * that was sent to the email address that they used to register.
+ *
+ * @param codeValidator used to test the input for valid formatting
+ * @param onResend function to call when the user clicks the 'resend code' button
+ * @param resendInstructions text to display ahead of the resend link/button
+ * @param resendLabel label for the resend link/button
+ * @param initialValue code used to pre-populate the field
+ *
+ * @category Component
+ */
+
+export const VerifyCodeScreenBase: React.FC<React.PropsWithChildren<React.PropsWithChildren<VerifyCodeScreenProps>>> = (
```

Comment:
There is no need of 2 React.PropsWithChildren. One React.PropsWithChildren should work.